Kernel/Net/E1000ENetworkAdapter.cpp

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2018-2021, Andreas Kling <kling@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/MACAddress.h>
+#include <Kernel/Debug.h>
+#include <Kernel/Net/E1000ENetworkAdapter.h>
+#include <Kernel/PCI/IDs.h>
+
+namespace Kernel {
+
+#define REG_EEPROM 0x0014
+
+static bool is_valid_device_id(u16 device_id)
+{
+    switch (device_id) {
+    case 0x10D3: // 82574
```

Comment:
An enum would be better, but I guess a comment works too...

Reply:
I'm trying to be consistent with what we have already, so if you look at `E1000NetworkAdapter.cpp` file, it's done the same there.